docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -557,3 +557,80 @@ JSON::
 }
 ----
 ====
+== Policy & Data Source URL formats
+
+The `policy` and `data` fields in the configuration represent the URI of the policy and data sources, respectively. The following formats are supported:
+
+=== Local File
+
+A local file path scheme, such as the following, may be utilized. 
+
+* `file://<path>:`
```

Comment:
Is the trailing : intentional here?

```suggestion
* `file://<path>`
```

maybe?

Reply:
Thanks for catching this. Initially theses were headings, but in editing things, they  were moved around. I'll remove the : char